HPC/WRF.md

```diff
@@ -0,0 +1,267 @@
+## WRF
+
+The Weather Research and Forecasting (WRF) model is one of the most used numerical weather prediction (NWP) system. WRF is used extensively for research and real-time forecasting with over 52,570 registered users in over 184 countries throughout the world as of Dec 2020. Large amount of computation resources are required for each simulation, especially for high resolution simulations. Compute and MPI performance are very important for WRF performance.
+
+## WRF correctness fix
+We have observed small numeric differences between WRF output models (v4.2.2) generated on Arm64 and intel platforms. The issue is caused by rounding error at last position (1ulp) from transcendental functions and several compiler optimization options. After identifying the root cause, we implemented [a patch](https://github.com/wrf-model/WRF/pull/1773) that allows bit-by-bit matching results on Graviton and intel processors for Ubuntu 18.04 with GCC 10.2 compiler. This patch works for both v4.2.2 and v4.5 (latest version). 
```

Comment:
won't it do the same for ubunt 22.04? if you call out 18.04 people might thnik you haev to use it.

Reply:
Also made a request for the parallel cluster team to make ubuntu22.04 and al2023 available